documentation/asciidoc/topics/proc_connecting_cli.adoc

```diff
@@ -0,0 +1,59 @@
+[id='connecting-cli_{context}']
+= Connecting the {brandname} CLI
+Use the command line interface (CLI) to connect to your {brandname} cluster and perform administrative operations.
```

Comment:
Is this meant to be in this PR?

I can't see where the file is linked.

Reply:
weird. it's a little hidden but here: https://github.com/infinispan/infinispan-operator/pull/905/files#diff-103028eb57da956f4d3a18e7f8b0d04f1bbc204c1f73ef09a364fedd466cbfd1R11 

In the current iteration of the docs we have instructions for using the CLI batch mode "manually" so the `Batch` CR supersedes that and I've condensed down all the CLI connection details into a single procedure.

Reply:
I had the file marked as "viewed" on GH :see_no_evil: